mm2src/coins/lightning.rs

```diff
@@ -361,7 +362,7 @@ impl LightningCoin {
         let mut total_open_channels: Vec<ChannelDetailsForRPC> =
             self.list_channels().await.into_iter().map(From::from).collect();
 
-        total_open_channels.sort_by(|a, b| a.rpc_channel_id.cmp(&b.rpc_channel_id));
+        total_open_channels.sort_by(|a, b| a.uuid.cmp(&b.uuid));
```

Comment:
Thanks for noticing this. Before this PR, `rpc_channel_id` was incremented when a new channel is opened, so this was like sorting by timestamp which is the desired behaviour when calling this RPC since it's similar to other get by filter functions. Now I think it's not needed with uuids randomly generated, but since there are no timestamp for when a channel is open, I can probably use [`short_channel_id`](https://github.com/lightningdevkit/rust-lightning/blob/558b2f2904cc555a52bd57edbe3be44d8cfce6be/lightning/src/ln/channelmanager.rs#L1022) instead since it represents the channel's funding tx place in the chain which is akin to a timestamp. Will fix it.

Reply:
Fixed